models/playlist.ts

```diff
@@ -17,4 +17,41 @@ module.exports.savePlaylist = function (name, songs, user_id, type, startdatetim
             return callback(err, result);
         }
     })
+};
+module.exports.saveOrUpdatePlaylist = function (name, songs, user_id, type, startdatetime, enddatetime, callback) {
+    var playlistFound = false;
+    pool.query("SELECT user_id ,name FROM playlists", function (err, result, fields) {
+        if (err) {
+            Logger.log('error', err);
+            callback(err, '');
+        }
+        else {
+            for (var i = 0; i < result.rows.length; i++) {
+                if (result.rows[i].user_id == user_id && result.rows[i].name == name) {
+                    var sql = 'DELETE FROM playlists WHERE user_id = $1 AND name = $2';
+                    pool.query(sql, [result.rows[i].user_id, result.rows[i].name], function (err, result) {
+                        if (err) {
+                            Logger.log('error', err);
+                            return callback(err, '');
+                        }
+                        else {
+                            playlistFound = true;
+                        }
+                    });
+                }
+            }
+            if (!playlistFound) {
```

Comment:
Alleen als die niet deleted is check je 'm. Want als ie daar door heen loopt en hij delete wel, is het altijd OF error OF true, (zie de if else statement...)

Dus wat gebeurt er als je hem wel delete?